README.md

```diff
@@ -5,10 +5,11 @@ Links:
 * [Adapter](src/myn/patterns/adapter)
 * [Builder](src/myn/patterns/builder)
 * [Composite](src/myn/patterns/composite) 
+* [Decorator](src/myn/patterns/decorator) 
```

Comment:
I like the idea behind sorting alphabetically

Reply:
thx)